app/javascript/mastodon/actions/notification_policies.ts

```diff
@@ -14,3 +16,7 @@ export const updateNotificationsPolicy = createDataLoadingThunk(
   'notificationPolicy/update',
   (policy: Partial<NotificationPolicy>) => apiUpdateNotificationsPolicy(policy),
 );
+
+export const decreasePendingNotificationsCount = createAction<number>(
+  'notification_policy/decrease_pending_notification_count',
```

Comment:
This should probably be `notificationPolicy/decreasePendingNotificationsCount` to be aligned with the action above and the naming of the action

Reply:
That was my ruby muscle-memory taking over :grimacing: